harness/run_eval.py

```
@@ -786,6 +805,126 @@ def main():
         else:
             model_specs = ["dummy"]
 
+    # Print configuration summary
+    def _get_env_or_default(key: str, default: str = "not set") -> str:
+        val = os.getenv(key)
+        return val if val else default
+
+    def _format_value(val: Any) -> str:
```

Ruff (0.14.7)

Dynamically typed expressions (typing.Any) are disallowed in `val`

(ANN401)